Samples/MNIST/UWP/cs/MNIST_Demo.csproj

```diff
@@ -85,7 +85,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
     <OutputPath>bin\x64\Release\</OutputPath>
-    <DefineConstants>TRACE;NETFX_CORE;WINDOWS_UWP</DefineConstants>
+    <DefineConstants>TRACE;NETFX_CORE;WINDOWS_UWP;USE_WINML_NUGET</DefineConstants>

```

Comment:
Can you add the NuGet package reference ? That way, the MNIST sample will build without users needing to add the NuGet package themselves through VS NuGet package manager.

<PackageReference Include="Microsoft.AI.MachineLearning">
      <Version>1.4.0</Version>
</PackageReference>

![image](https://user-images.githubusercontent.com/16694211/93649849-c6f26480-f9c1-11ea-80a9-9c8a8dfa8ab7.png)

Reply:
Yes, I'll do it when the Nuget with store support is released.